web-client/src/presenter/actions/setValidationAlertErrorsAction.test.ts

```diff
@@ -140,4 +141,23 @@ describe('setValidationAlertErrors', () => {
       messages: ['second nested', 'first nested'],
     });
   });
+
+  it('state.alertError sets scrollToErrorNotification appropriately', async () => {
```

Comment:
nit: this doesn't really read like a sentence ("it does x [when y]") the same way our other test cases do.

Reply:
You're right--I will update! Thanks :)